pkgs/development/tools/analysis/rizin/default.nix

```diff
@@ -0,0 +1,52 @@
+{ stdenv
+, fetchFromGitHub
+, ninja, meson
+, libzip, openssl, xxHash, zlib
+}:
+
+stdenv.mkDerivation {
+  pname = "rizin";
+  version = "unstable-2021-01-13";
+
+  src = fetchFromGitHub {
+    owner = "rizinorg";
+    repo = "rizin";
+    rev = "ca2471ae9596176c656f065ed74c480f13784e1b";
+    sha256 = "0qigy1px0jy74c5ig73dc2fqjcy6vcy76i25dx9r3as6zfpkkaxj";
+  };
```

Comment:
They still rely on git to set versions etc:

https://github.com/rizinorg/rizin/blob/d4964454765539c61cce7f3f845f8d7fe3a3f076/Makefile#L60

In our other radare2 build we generate those automatically.